src/pet.js

```diff
@@ -0,0 +1,82 @@
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_HUNGER = 0;
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = MINIMUM_HUNGER;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.fitness > 0 && this.hunger < 10 && this.age < 30;
+    }
+}
+
+Pet.prototype.growUp = function() {
```

Comment:
What would happen in the instance of the `growUp` function being called multiple0 times? Should there be checks here to make sure the values for age hunger and fitness do not go past their limits?